packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -38,6 +39,22 @@ class OpenAPIBuilder implements Builder {
 
   @override
   Future<void> build(final BuildStep buildStep) async {
+    try {
+      final result = await helperVersionCheck(buildStep);
+
+      if (result.messages.isNotEmpty) {
+        log.warning(result.messages);
+      }
+
+      if (result.hasFatal) {
+        log.warning(result.messages);
+        return;
+      }
```

Comment:
Shouldn't this throw an exception? Otherwise we also won't notice it in our CI.

Reply:
I thought returning was enough but you are right.
Thanks for catching